`make update` produces Error 127

To reproduce on macOS Sierra 10.12.6

```bash
# in /Users/pokus/golang/src/github.com/elastic/apm-server
$ make update
bash: virtualenv: command not found
make: *** [python-env] Error 127
```

Although `$ make` worked and produced the `apm-server`.

If this is an issue with python I have `python2 -> ../Cellar/python/2.7.13_1/bin/python2` in `/usr/local/bin` and `python2.7 -> ../../System/Library/Frameworks/Python.framework/Versions/2.7/bin/python2.7` in `/usr/bin`.

`$PATH` is `/Users/pokus/.nvm/versions/node/v6.11.3/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/opt/go/libexec/bin`

 Not sure what to do.

**edit:** am I missing `virtualenv `?

--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ You also need to create all files needed by the APM Server by running the additi
 ```
 make update
 ```
-
+Note that this requires to have `virtualenv` installed.
 
 ### Run
 

